src/phoenix/experimental/evals/models/bedrock.py

```diff
@@ -142,7 +143,23 @@ def _generate_with_retry(self, **kwargs: Any) -> Any:
         @self.retry
         @self._rate_limiter.limit
         def _completion_with_retry(**kwargs: Any) -> Any:
-            return self.client.invoke_model(**kwargs)
+            try:
+                return self.client.invoke_model(**kwargs)
+            except Exception as e:
+                exception_message = e.args[0]
+                if not exception_message:
+                    raise e
```

Comment:
Is this a check for `None`? Doesn't look like we're doing it elsewhere, may not be needed if the message is guaranteed to exist?

Reply:
I combined the existence check to the string matching condition in the other models